import { CustomError } from "./CustomError";

export class PostNotFound extends CustomError {
    constructor() {
        super("Post n√£o encontrado!", 404)
    }
} 